src/core_ocean/shared/mpas_ocn_tracer_hmix_redi.F

```diff
@@ -188,6 +189,7 @@ subroutine ocn_tracer_hmix_Redi_tend(meshPool, scratchPool, layerThickness, laye
         enddo
         rediKappaCell(iCell) = rediKappaCell(iCell) / areaCell(iCell)
       enddo
+      !$omp end do
```

Comment:
@mark-petersen testing by @jonbob confirmed by my own testing shows that if we comment out L183 and L192 PET passes and with the lines the run fails.  It doesn't make sense why.  Do you have any thoughts?

Reply:
I agree that doesn't make any sense. It must be something else, and this just reveals it.